developers/weaviate/more-resources/migration/weaviate-1-30.md

```diff
@@ -0,0 +1,181 @@
+---
+title: 1.30 (BlockMax WAND)
+sidebar_position: 9
+image: og/docs/more-resources.jpg
+# tags: ['migration']
+---
+
+# Weaviate 1.30 migration guide
```

Comment:
Hey @g-despot - this guide is good. 

But imo this is specific to BlockMax WAND index migration, less of a general migration guide. It does not provide for example steps on how to update the Weaviate version, migrate any other data. So I suggest framing this as a BlockMax WAND index migration guide.